queries.sql

```diff
@@ -1,3 +1,18 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+/* Find all animals whose name ends in "mon". */
+SELECT * FROM animals WHERE name LIKE '%mon';
+/* List the name of all animals born between 2016 and 2019.*/
+SELECT DISTINCT name FROM animals WHERE date_of_birth BETWEEN '2016-01-01' and '2020-12-30'  
```

Comment:
* Kindly write the correct query for this quest `List the name of all animals born between 2016 and 2019.` as your current query isn't correct, it should search between 2016 and 2019